### PR TITLE
Buffer movement using h,j,k,l

### LINKED ISSUE #146: Why default to uppercase letters for buffer movement?
Issue Type: <b>Feature Request</b>

Hello,
I was to move a buffer between tabs/groups and I noticed that all the motion letters (h,j,ik,l) should be typed uppercase. Why? There is no other binding under that submenu using then, then why not use the normal letters as default? It will be much simpler and faster to type.

Extension version: 0.8.5
VS Code version: Code 1.51.1 (e5a624b788d92b8d34d1392e4c4d9789406efe8f, 2020-11-11T01:11:34.018Z)
OS version: Darwin x64 20.1.0

### PR DESCRIPTION
**Pull request by danielo515**
Fixes #146

This adds buffer handling using lowercase h,j,k,l keys

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -335,6 +335,30 @@
 										"name": "Show all buffers in active window",
 										"type": "command",
 										"command": "workbench.action.showEditorsInActiveGroup"
+									},
+																		{
```
Comment (stevenguh) [inline package.json:339]: There are extra spaces in this line

**Review by stevenguh — CHANGES_REQUESTED** [review]
We sort the keys by lower and alpha order and then upper case, can you move the keys to the right order? Also can you update CHANGELOG as well

**Comment by danielo515** [comment]
> We sort the keys by lower and alpha order and then upper case, can you move the keys to the right order? Also can you update CHANGELOG as well. Thank you :)

Fixed the keys order.
How do you update the changelog? You do it manually or is there any CLI that you use?
Should I bump a minor version then?

File: package.json [hunk 2]
```diff
@@ -2867,4 +2891,4 @@
 		"jsonc-parser": "^2.3.0",
 		"lodash": "^4.17.19"
 	}
-}
+}
```
Comment (stevenguh) [inline package.json:2894]: There shouldn't be a change here. I think you took out the last new line character.

**Comment by stevenguh** [comment]
> Fixed the keys order.
> How do you update the changelog? You do it manually or is there any CLI that you use?
> Should I bump a minor version then?
>
> -- EDIT: --
> I think I did it correctly, can you check again?

Thank you for fixing the order and adding the CHANGELOG. Yeah we manually add the unreleased change in the change log, and I think you did it :) Once you fixed the bogus new line character, I will merge this.

File: package.json (hunk 2 above)
Comment (danielo515) [reply package.json]: fixed

**Pull request merged**